### PR TITLE
update Avatars

### PR DESCRIPTION
**Pull request by thesahindia**
### Details

Added different Avatar for #announce and #admins room
Changed the Avatar of rooms and deleted rooms
### Fixed Issues

$ https://github.com/Expensify/App/issues/8147
### Tests | QA Steps

1. Search for a default room - either #announce or #admins
2. Verify that Avatar for #announce room is a megaphone
3. Verify that Avatar for #admins room is shield
4. Delete the workspace and navigate to deleted room and check that Avatar is a bin with blue background
5. Click on the fab icon and tap on new room > fill the details > check that the Avatar is a couch with blue background 
- [x] Verify that no errors appear in the JS console

#### Contributor (PR Author) Checklist
- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [ ] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] iOS / native
    - [x] Android / native
    - [x] iOS / Safari
    - [x] Android / Chrome
    - [x] MacOS / Chrome
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there’s a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. “toggleReport” and not “onIconClick”)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct english, and explained “why” the code was doing something instead of only explaining “what” the code was doing.
    - [ ] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [ ] I verified any copy / text that was added to the app is correct english and approved by marketing by tagging the marketing team on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named “index.js”. All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY
- [x] I verified any variables that can be defined as constants (ie. in CONST.js) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] Any functional components have the `displayName` property
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] Any internal methods are bound to “this” properly so there are no scoping issues
    - [ ] Any internal methods bound to “this” are necessary to be bound
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose and it is
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn’t already exist
    - [ ] The style can’t be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function
(i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)

#### PR Reviewer Checklist
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] iOS / native
    - [ ] Android / native
    - [ ] iOS / Safari
    - [ ] Android / Chrome
    - [ ] MacOS / Chrome
    - [ ] MacOS / Desktop
- [x] I verified there are no console errors (if there’s a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. “toggleReport” and not “onIconClick”).
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct english, and explained “why” the code was doing something instead of only explaining “what” the code was doing.
    - [ ] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [ ] I verified any copy / text that was added to the app is correct english and approved by marketing by tagging the marketing team on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named “index.js”. All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components are not impacted by changes in this PR (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY
- [x] I verified any variables that can be defined as constants (ie. in CONST.js) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] Any functional components have the `displayName` property
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] Any internal methods are bound to “this” properly so there are no scoping issues
    - [ ] Any internal methods bound to “this” are necessary to be bound
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose and it is broken down into smaller components in order to separate concerns and functions
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn’t already exist
    - [ ] The style can’t be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function
(i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)

#### Web

https://user-images.githubusercontent.com/83179295/158691628-a1fe7add-5c09-4935-b2c7-a8163117e9e8.mov

#### Mobile Web

https://user-images.githubusercontent.com/83179295/158691658-53411905-28f2-4129-86b7-d32a92131479.mov

#### Desktop

https://user-images.githubusercontent.com/83179295/158691861-5dc0c1f4-6816-416a-82c0-9edbc9679299.mov

#### iOS

https://user-images.githubusercontent.com/83179295/158691898-a1374d78-cc83-45db-bbdd-9feac3e16cca.mov

#### Android

https://user-images.githubusercontent.com/83179295/158691831-8486c608-768a-47e6-ac9c-dd8edec66900.mov

### CODE REVIEW

**Comment by thesahindia** [comment]
In the OP the expected result was to change the icons for #announce and #admins room but the zip file that @shawnborton had provided also contains a SVG for private room. So, do we wanna show a different Avatar for private rooms? If yes then should I add a function which checks if the room is both user-made and private? 
We have to check if it's user-made because #announce and #admins both are private rooms and if we change the Avatar for all the private rooms then their Avatar will also be changed

cc: @stitesExpensify @shawnborton

**Comment by shawnborton** [comment]
Let's not worry about the private room functionality for now, we can follow up with that in a separate project.

File: src/components/Icon/Expensicons.js [hunk 1]
```diff
@@ -72,10 +72,14 @@ import Wallet from '../../../assets/images/wallet.svg';
 import Workspace from '../../../assets/images/workspace-default-avatar.svg';
 import ActiveRoomAvatar from '../../../assets/images/avatars/room.svg';
 import DeletedRoomAvatar from '../../../assets/images/avatars/deleted-room.svg';
+import AdminsRoomAvatar from '../../../assets/images/avatars/admins-room.svg';
+import AnnounceRoomAvatar from '../../../assets/images/avatars/announce-room.svg';
 
 export {
     ActiveRoomAvatar,
+    AdminsRoomAvatar,
```
Comment (rushatgabhane) [inline src/components/Icon/Expensicons.js:80]: Is it just me or AdminsRoomAvatar sounds kinda weird?
How about `AdminRoomAvatar`? @thesahindia

Comment (thesahindia) [reply src/components/Icon/Expensicons.js:80]: Yeah I was going to use that but it's for #admins room so I thought AdminsRoomAvatar will be correct.

Comment (rushatgabhane) [reply src/components/Icon/Expensicons.js:80]: No problem, let's change it to AdminRoomAvatar everywhere for now

**Review by rushatgabhane — CHANGES_REQUESTED** [review]
@thesahindia to be pedantic

**Comment by thesahindia** [comment]
I have made the suggested changes, it's ready for the review now.

**Review by rushatgabhane — APPROVED** [review]
Thanks for the changes @thesahindia 

@stitesExpensify LGTM! 🎉

#### PR Reviewer Checklist

- [x]  I verified the correct issue is linked in the ### Fixed Issues section above
- [x]  I verified testing steps are clear and they cover the changes made in this PR
  - [x]  I verified the steps for local testing are in the Tests section
  - [x]  I verified the steps for Staging and/or Production testing are in the QA steps section
  - [x]  I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
  - [x]  I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] iOS / native
    - [x] Android / native
    - [x] iOS / Safari
    - [x] Android / Chrome
    - [x] MacOS / Chrome
    - [x] MacOS / Desktop
- [x] I verified there are no console errors related to changes in this PR
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
  - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. “toggleReport” and not “onIconClick”).
  - [x]  I verified that comments were added to code that is not self explanatory
  - [x]  I verified that any new or modified comments were clear, correct english, and explained “why” the code was doing something instead of only explaining “what” the code was doing.
  - [x]  I verified any copy / text shown in the product was added in all src/languages/* files
  - [x]  I verified any copy / text that was added to the app is correct english and approved by marketing by tagging the marketing team on the original GH to get the correct copy.
  - [x]  I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named “index.js”. All platform-specific files are named for the platform the code supports as outlined in the README.
  - [x]  I verified the JSDocs style guidelines (in [STYLE.md](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs)) were followed
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components are not impacted by changes in this PR (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified the UI performance was not affected (the performance is the same than `main` branch)
- [x] If a new component is created I verified that 
  - [x]  A similar component doesn't exist in the codebase
  - [x]  All props are defined accurately and each prop has a /** comment above it */
  - [x]  Any functional components have the displayName property
  - [x]  The file is named correctly
  - [x]  The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
  - [x]  The only data being stored in the state is data necessary for rendering and nothing else
  - [x]  Any internal methods are bound to “this” properly so there are no scoping issues
  - [x]  Any internal methods bound to “this” are necessary to be bound
  - [x]  All JSX used for rendering exists in the render method
  - [x]  The component has the minimum amount of code necessary for its purpose and it is broken down into smaller components in order to separate concerns and functions
 - [x]  If a new CSS style is added I verified that:
    - [x]  A similar style doesn’t already exist
    - [x]  The style can’t be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function
    - [x] (i.e. StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)

**Review by stitesExpensify — APPROVED** [review]

**Pull request merged**